src/components/widgets/explore-capabilities.tsx

```diff
@@ -127,18 +129,17 @@ const ExploreCapabilities: React.FunctionComponent = () => {
                   {drawerData[activeItem].body}
                 </Text>
               </TextContent>
-            </CardBody>
-            <CardFooter className="pf-v5-u-p-0">
               <Button
                 component="a"
                 size="lg"
                 href={drawerData[activeItem].url}
                 className="pf-m-danger pf-v5-u-mb-sm"
                 ouiaId={drawerData[activeItem].ouiaId}
+                style={{ marginTop: '1rem' }}
```

Comment:
Please use CSS instead of inline styling.

Reply:
@Hyperkid123, thanks, edited in the last push